pkg/provider/aws/resources/eks.go

```diff
@@ -102,6 +102,36 @@ func CreateEksCluster(appName string, clusterName string, subnets []*Subnet, sec
 		dag.AddDependency(nodeGroup, s)
 		dag.AddDependency(profile, s)
 	}
+
+	for _, addOn := range createAddOns(clusterName, references) {
+		dag.AddResource(addOn)
+		dag.AddDependency(addOn, nodeGroup)
```

Comment:
we have the dependsOn in the helm chart factory right? Pretty sure we do, but if not will need to add it

Reply:
yep, we do